toolbox/db/db_get.m

```diff
@@ -201,63 +207,29 @@
         end
         varargout{1} = sql_query(sqlConn, 'select', 'subject', '*', [], addQuery);
 
+%% ==== SUBJECTS COUNT ====
+    % nSubjects = db_get('SubjectCount')
+    case 'SubjectCount'
+        if isempty(GlobalData.DataBase.iProtocol) || (GlobalData.DataBase.iProtocol == 0)
+            varargout{1} = 0;
+            return;
+        end
```

Comment:
I'm not sure this part making sure a protocol is loaded is relevant to be included in `db_get()`.